src/it/projects/flatten-dependency-all-depmgnt-provided-transitive/pom.xml

```diff
@@ -0,0 +1,37 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.codehaus.mojo.flatten.its</groupId>
+    <artifactId>flatten-dependency-all-depmgnt-provided-transitive</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.codehaus.mojo.flatten.its</groupId>
+                <artifactId>dep</artifactId>
+                <version>1.1</version>
+                <scope>provided</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.codehaus.mojo.flatten.its</groupId>
+            <artifactId>parent-deps</artifactId>
```

Comment:
I'm confused. Why is this called parent-deps when its not the parent of this pom? Consider renaming some things.

Reply:
I believe it was called parent-deps because it was the parent of 'dep'. Currently there's a test artifact parent-deps:1.0 already within the 'mrm' folder used for testing. 
In the PR, we add another test artifact parent-deps:1.1 that holds the same structure as the original parent-deps:1.0, but uses 'dep' with scope 'provided' instead, and following the conventions that were present with the original parent-deps:1.0.